test/topology_custom/test_mv_fail_updates.py

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import asyncio
+import pytest
+import time
+from test.topology.conftest import skip_mode
+from test.pylib.manager_client import ManagerClient
+
+from cassandra.cluster import ConsistencyLevel  # type: ignore
+from cassandra.query import SimpleStatement  # type: ignore
+
+
+async def wait_for_views(cql, node_count):
+    deadline = time.time() + 120
+    while time.time() < deadline:
+        done = await cql.run_async(f"SELECT COUNT(*) FROM system_distributed.view_build_status WHERE status = 'SUCCESS' ALLOW FILTERING")
+        if done[0][0] == node_count:
+            return
+        else:
+            time.sleep(0.2)
+    raise Exception("Timeout waiting for views to build")
+
+
+# This test makes sure that even if the view updates generation is failing on a node,
+# the reads can still succeed, as long as we can reach quorum.
```

Comment:
Also, it's not surprising that reads succeed even if updates to one node fails...
Quite the contrary - it's surprising that in some cases the read fails because read-repair fails due to write failures - but neither this comment nor the commit message say that this is what you're trying to test.